otel-extensions/src/main/java/org/hypertrace/agent/otel/extensions/config/DataCaptureConfigImpl.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright The Hypertrace Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.hypertrace.agent.otel.extensions.config;
+
+import com.google.auto.service.AutoService;
+import com.google.protobuf.StringValue;
+import java.util.List;
+import org.hypertrace.agent.config.v1.Config;
+import org.hypertrace.agent.core.config.DataCaptureConfig;
+
+/**
+ * An implementation of the DataCaptureConfig interface, which can provide the names of the content
+ * types that can be captured.
+ */
+@AutoService(DataCaptureConfig.class)
```

Comment:
Why can't the default allowed content types be just a list of strings?

Reply:
I followed the precedent of defining implementations for the portions of the configuration that are required by the Java agent.  The Java agent uses Java services to perform the initialization of config data; I merely continued the practice, for consistency.

Reply:
I see. Ok.